plugin-server/src/main/utils.ts

```diff
@@ -39,8 +35,6 @@ export async function runInstrumentedFunction<T, E>({
         Sentry.captureException(error, { tags: { team_id: teamId } })
         throw error
     } finally {
-        server.statsd?.increment(`${statsKey}_total`)
-        server.statsd?.timing(statsKey, timer)
```

Comment:
I don't think we have prom metrics for these, could you add them?

Reply:
This is the end call above

Reply:
Will update dashboards though

Reply:
can I get those dash updates? Time spent is quite a nice metric to use